requirements.txt

```diff
@@ -1 +1 @@
--e .
```

Comment:
weird change, is it whitespace / new line? lets take it out anyway. and squash commits?

Reply:
ah right, the newline got lost there, thanks, restored now